src/Documentation/Markdown/Markdown.js

```diff
@@ -61,7 +61,7 @@ const HtmlRenderer = props => {
 const CodeBlock = ({ value, language }) => {
   const dvcStyle = Object.assign({}, docco)
   dvcStyle['hljs-comment'] = { color: '#999' }
-  dvcStyle['hljs-meta'] = { color: '#333', fontSize: '14px' }
+  dvcStyle['hljs-meta'] = { color: '#333', fontSize: '14px', paddingLeft: '8em' }
```

Comment:
I think you should try something like this: `dvcStyle['hljs']['padding']='0.5em 0.5em 0.5em 8em'` - it's a more general solution that would apply padding to the whole code block